### ISSUE TITLE
MongoDB for mongodb-datastore loses data when being shut down

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn/keptn#1360]
Happened on internal-demo-cluster with Keptn 0.6.0 when draining a node.

How to reproduce:

1. Install Keptn 
2. Onboard a project and send new artifacts
3. Open Keptn's Bridge

4. Stop the mongodb pod
```console
kubectl get pods -n keptn-datastore
kubectl -n keptn-datastore delete pod mongodb-758f7975c6-7ffnc
```

5. Verify the pod has been restarted:
```console
kubectl get pods -n keptn-datastore
```

6. Look at Keptn's Bridge - all events are gone

**Comment by christian-kreuzberger-dtx** [comment]
The issue is that the mountpath for the volume is wrong:

https://github.com/keptn/keptn/blob/e789627d0e7d254689040d4cc25c90387c235927/installer/manifests/logging/mongodb-k8s/deployment.yaml#L35-L37

It should be `/var/lib/mongodb/data`

**Comment by christian-kreuzberger-dtx** [comment]
I discovered that there are currently two major issues with our mongodb deployment.

1. `/data/db` is the wrong mountpath (though this seems to be the recommended way to do it with Mongodb in Kubernetes)
2. the persistent volume is everything but persistent, the data is cleaned every time we restart the pod.

**Comment by christian-kreuzberger-dtx** [comment]
I fixed this in PR #1362 by doing the following three things:

1. The volume needs to be set to `/var/lib/mongodb/data`
2. The volume needs to have permissions for the user group set by using
```yaml
      securityContext:
        fsGroup: 184
```
3. The size of the volume needs to be increased from 100 MB to at least 1 GB
